api/v1alpha1/nodehealthcheck_types.go

```diff
@@ -289,6 +289,11 @@ type Remediation struct {
 	//+optional
 	//+operator-sdk:csv:customresourcedefinitions:type=status
 	TimedOut *metav1.Time `json:"timedOut,omitempty"`
+
+	// TemplateName is required when using several templates of the same kind
```

Comment:
IIUC we use [GetCurrentTemplateWithTimeout](https://github.com/medik8s/node-healthcheck-operator/blob/main/controllers/resources/templates.go#L37) method in order to figure out which template to use next.
In order to do so we compare  `*remediationv1alpha1.Remediation` to the template, since now a remediationv1alpha1.Remediation gvk might match multiple templates of the same kind, we need a definitive indicator  in order to be able to match it to the correct template.

Reply:
makes sense, thanks!